templates/base/server/.rtag/methods.ts.hbs

```diff
@@ -7,16 +7,21 @@ import {
   {{/each}}
 } from "./types";
 
+export interface Context {
+  randInt(): number;
```

Comment:
What if I have a function that requires more than one or two independently random events? For example, if I want to implement a perfectly random deck shuffle.

Reply:
you can just call the function multiple times (it will return different values)